vignettes/setup/setup.R

```diff
@@ -19,12 +19,14 @@ knit_print.grViz <- function(x, ...) {
     knitr::asis_output()
 }
 
-# If input loads dm...
+# If input loads dm or tidyverse, we load it here to omit warnings
 input <- readLines(knitr::current_input())
 if (rlang::has_length(grep("^library[(]dm[)]", input))) {
-  # we load it here to omit warnings
   library(dm)
 }
+if (rlang::has_length(grep("^library[(]tidyverse[)]", input))) {
```

Comment:
so does this work if someone writes `library("tidyverse")`, as old fashioned as it is?

Reply:
It didn't work. But I've changed that.

Reply:
Actually, I am not sure if it makes sense to support this. The coding style across our vignettes should be consistent. This means that we should be consistently writing `library(tidyverse)`, and if someone uses `library("tidyverse")`, this should be asked to be changed during the code review.

Sounds reasonable?

Reply:
Okay, now we support both ways to specify both libraries, so this is resolved.